$DragonFly: src/gnu/usr.bin/cc3/cc_int/patches/Attic/config,elfos.h.patch,v 1.1 2004/05/15 16:17:57 joerg Exp $
Index: config/elfos.h
===================================================================
RCS file: /home/joerg/wd/repo/dragonflybsd/src/contrib/gcc-3.3/gcc/config/elfos.h,v
retrieving revision 1.1
diff -u -r1.1 elfos.h
--- config/elfos.h	2 Feb 2004 23:09:50 -0000	1.1
+++ config/elfos.h	14 May 2004 14:47:56 -0000
@@ -73,7 +73,7 @@
    default to dwarf2.  */
 
 #ifndef PREFERRED_DEBUGGING_TYPE
-#define PREFERRED_DEBUGGING_TYPE DWARF2_DEBUG
+#define PREFERRED_DEBUGGING_TYPE DBX_DEBUG
 #endif
 
 /* All SVR4 targets use the ELF object file format.  */
